.github/workflows/docs.yml

```diff
@@ -0,0 +1,58 @@
+name: Build and deploy Jekyll documentation site
+
+on:
+  pull_request:
+    branches:
+      - master
+
+env:
+  RUBY_VERSION: 2.7
+
+jobs:
+  deploy_docs:
+    if: "!contains(github.event.commits[0].message, '[ci skip]')"
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+      - name: Setup cache for Bundler
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-bundler-${{ env.RUBY_VERSION }}-${{ hashFiles('Gemfile') }}-${{ hashFiles('jekyll.gemspec') }}
+          restore-keys:
+            - ${{ runner.os }}-bundler-${{ env.RUBY_VERSION }}-${{ hashFiles('Gemfile') }}-
+            - ${{ runner.os }}-bundler-${{ env.RUBY_VERSION }}-
+      - name: Set up dependencies
+        run: |
+          gem update --system --no-document
+          gem update bundler --no-document
+          bundle install --path=vendor/bundle --jobs 4 --retry 3
+          bundle clean
+      - name: Build site
+        run: bundle exec jekyll build --source docs --destination docs/_site --verbose --trace
+        env:
+          JEKYLL_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What's the significance of this line? There is no commit steps here like in `REMOTE_REPO` setup below.

Does this have to do with `github-pages` plugin and getting GH metadata? I know that plugin uses `GITHUB_TOKEN` and gives build warnings locally if that is not set, so wondering if `JEKYLL_PAT` works for that?

Reply:
Looks like I mixed things up. I can't recall when the knowledge of `JEKYLL_PAT` emerged in my head, but I just went through stuff again and am now confident it's a mistake. I've changed this into `JEKYLL_GITHUB_TOKEN` which is wanted by the `jekyll-github-metadata` plugin.